examples/drivers/windows/otLwf/radio.c

```diff
@@ -759,6 +759,13 @@ ThreadError otPlatRadioEnergyScan(_In_ otInstance *otCtx, uint8_t aScanChannel,
     return NT_SUCCESS(status) ? kThreadError_None : kThreadError_Failed;
 }
 
+void otPlatRadioSetDefaultTxPower(otInstance *aInstance, int8_t aPower)
+{
+    // TODO: Create a proper implementation for this driver.
+    (void)aInstance;
```

Comment:
This will be called as part of NcpBase::SetPropertyHandler_PHY_TX_POWER in response to spinel property SPINEL_PROP_PHY_TX_POWER.  Is that not sufficient?